libraries/AP_Common/Bitmask.h

```diff
@@ -44,29 +60,48 @@ class Bitmask {
 
     // clear all bits
     void clearall(void) {
-        for (uint16_t i=0; i<numwords; i++) {
-            bits[i] = 0;
-        }
+        memset(bits, 0, numwords);
```

Comment:
I think this is wrong.  bits is an int32_t but we are only setting the first numwords bytes of it.  Shouldn't this be 
memset(bits, 0, sizeof(bits))

Reply:
it is a bug, but sizeof(bits) would be worse, as that is a pointer.
The original code was correct, or you could do numwords*sizeof(bits[0])

Reply:
hahahaha - of course.  Thanks Tridge.

Reply:
@squilter, can you check this one?  If it's just a matter of reverting it I can do that.. or you can just make another commit.  whatever works.